packages/core-interfaces/src/core-database/database-repository/transactions-repository.ts

```diff
@@ -37,9 +38,20 @@ export interface ITransactionsRepository extends IRepository {
         totalAmount: Bignum
     }>;
 
+    /* TODO: Maybe consolidate this with 'statistics()'. How are they used, and why separate implementations? */
```

Comment:
The `statistics` method is used to verify the integrity of the database. It's only used during the boot process of the node because the query is very expensive.

Reply:
ah ok.